src/config.c

```diff
@@ -2611,7 +2625,7 @@ standardConfig configs[] = {
     createStringConfig("ignore-warnings", NULL, MODIFIABLE_CONFIG, ALLOW_EMPTY_STRING, server.ignore_warnings, "", NULL, NULL),
     createStringConfig("proc-title-template", NULL, MODIFIABLE_CONFIG, ALLOW_EMPTY_STRING, server.proc_title_template, CONFIG_DEFAULT_PROC_TITLE_TEMPLATE, isValidProcTitleTemplate, updateProcTitleTemplate),
     createStringConfig("bind-source-addr", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.bind_source_addr, NULL, NULL, NULL),
-    createStringConfig("logfile", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.logfile, "", NULL, NULL),
+    createStringConfig("logfile", NULL, IMMUTABLE_CONFIG | PROTECTED_CONFIG, ALLOW_EMPTY_STRING, server.logfile, "", NULL, NULL),
```

Comment:
This is already immutable. If we're being defense, perhaps we should also protect the cluster-config-file and the aclfile. Otherwise, let's drop this.

Reply:
we didn't bother to protect appendfilename since it's mutable, we should drop this one too,.